packages/react-templates/src/components/Select/SelectSimple.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Select, SelectList } from '@patternfly/react-core/dist/esm/components/Select';
+import { MenuToggle, MenuToggleElement } from '@patternfly/react-core/dist/esm/components/MenuToggle';
+
+export interface SelectSimpleProps {
+  /** @hide Forwarded ref */
+  innerRef?: React.Ref<any>;
+  /** Initial options of the select. */
+  initialOptions?: React.ReactElement[];
```

Comment:
Probably want to decide what exactly is passed to this prop, either an array of SelectOptions or an array of objects like in the [checkbox template PR](https://github.com/patternfly/patternfly-react/pull/10159/commits/27a8be04ff005b194773305800b2f8779b7fb6fb#diff-f739789cccb17b7e51546f4e5b4e8c9c470a047ade976321a276e47ca3e8ddaa). Keeping the type consistent would probably be less confusing for consumers if they're using both templates.

Reply:
IMO I think we should just launch all of these in beta and come back and make the API's consistent once we know where we land on all of our planned templates.

Reply:
Updated to SelectOptionProps[] but we can revisit props after other templates have gone in to review consistency.